src/main/java/com/clanki/parser/Parser.java

```diff
@@ -94,13 +97,16 @@ public static ReviewCommand getReviewCommand(ParsedInput parsedInput) {
         return new ReviewCommand();
     }
 
-    public static UpdateCommand getUpdateCommand(ParsedInput parsedInput) {
+    public static UpdateCommand getUpdateCommand(ParsedInput parsedInput) throws NoQueryInInputException {
+        String query = parsedInput.getBody();
```

Comment:
I think the current implementation may not follow what is said in the user guide, either change the implementation or change the user guide